tests/Api/JsonApiTestCase.php

```diff
@@ -13,47 +13,24 @@
 
 namespace Sylius\Tests\Api;
 
-use ApiTestCase\JsonApiTestCase as BaseJsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
 
-abstract class JsonApiTestCase extends BaseJsonApiTestCase
+class JsonApiTestCase extends AbstractJsonApiTestCase
```

Comment:
I would be for removing this class as you've changed its name and moving the below test to another class with a more appropriate name.